.cypress/integration/1_event_analytics.spec.js

```diff
@@ -45,12 +45,11 @@ describe('Adding sample data and visualization', () => {
 describe('Has working breadcrumbs', () => {
   it('Redirect to correct page on breadcrumb click', () => {
     landOnEventExplorer();
-    cy.get('.euiBreadcrumb[href="#/event_analytics/explorer"]').contains('Explorer').click();
-    cy.get('[data-test-subj="searchAutocompleteTextArea"]').should('exist');
-    cy.get('.euiBreadcrumb[href="#/event_analytics"]').contains('Event analytics').click();
-    cy.get('.euiTitle').contains('Event analytics').should('exist');
-    cy.get('.euiBreadcrumb[href="observability-dashboards#/"]').contains('Observability').click();
-    cy.get('.euiTitle').contains('Event analytics').should('exist');
+    suppressResizeObserverIssue();
+    cy.wait(delay);
```

Comment:
We want to avoid any explicit wait as it is considered as anti-pattern which may lead to flaky or inefficient tests. You may add timeout to below 'get' method to be something like ```cy.get('.euiBreadcrumb[href="observability-logs#/"]', {timeout: xxx}).click();```it will retry and wait for the element to appear in DOM.